packages/vue-3/src/Editor.ts

```diff
@@ -74,19 +74,39 @@ export class Editor extends CoreEditor {
     plugin: Plugin,
     handlePlugins?: (newPlugin: Plugin, plugins: Plugin[]) => Plugin[],
   ): void {
-    super.registerPlugin(plugin, handlePlugins)
+    const plugins = isFunction(handlePlugins)
+      ? handlePlugins(plugin, [...this.state.plugins])
+      : [...this.state.plugins, plugin]
+
+    const state = this.state.reconfigure({ plugins })
+
+    this.view.updateState(state)
+
     if (this.reactiveState) {
-      this.reactiveState.value = this.view.state
+      this.reactiveState.value = state
```

Comment:
This is the only real difference from what it was doing before no? Why do we need to duplicate what the editor class is doing?

The same was done for the unregistering, so I'd like to understand better what the actual problem was

Reply:
You are right. I’ve added more context to the PR's description. I'm using the reconfigured state directly since `this.view.state` is not updated at that point.